rules/opentitan/cc.bzl

```diff
@@ -339,6 +339,7 @@ opentitan_binary = rv_rule(
             doc = "List of execution environments for this target.",
         ),
         "_cc_toolchain": attr.label(default = Label("@bazel_tools//tools/cpp:current_cc_toolchain")),
+        "skip_signing": attr.bool(default = False),
```

Comment:
Maybe use the `kind` attribute in `common_binary_attrs` would be more appropriate? There is currently `ram`, `rom` and `flash`. This could be a new one. Maybe @cfrantz can weight in on that?